pandas/core/indexes/base.py

```diff
@@ -1893,7 +1893,11 @@ def _get_level_number(self, level) -> int:
         return 0
 
     def sortlevel(
-        self, level=None, ascending: bool | list[bool] = True, sort_remaining=None
+        self,
+        level=None,
+        ascending: bool | list[bool] = True,
+        sort_remaining=None,
+        na_position: str_t = "first",
```

Comment:
@phofl - any reason to default to first here? With a quick grep I'm seeing we default to last everywhere else.

Reply:
Pretty sure that this was done for backwards compatibility reasons. Does this break anything?

Reply:
Nope - makes sense. I'll experiment with changing the default and see what breaks; depending on that may put up an issue to deprecate.

Reply:
No objections to deprecate. IIRC this is used internally a bunch, so deprecating might be tricky

Reply:
Ah - I meant just deprecating the default and changing it to "last" to be consistent with other sorting methods. We can still specify "first" internally so hopefully it will be straight forward unless I'm missing something.

Reply:
Ah got you. No that makes sense